core/app/modules/import-export/assets/js/pages/import/import-process/import-process.js

```diff
@@ -57,7 +57,19 @@ export default function ImportProcess() {
 			importContext.dispatch( { type: 'SET_FILE', payload: null } );
 
 			navigateToMainScreen();
-		};
+		},
+		eventTracking = ( action, event, source, step, eventType = null ) => elementorCommon.events.eventTracking(
```

Comment:
why some of them use $e and the others use elementorCommon?

Reply:
This draft PR was pushed in order to allow Mati to check some of the events. I have implemented the `elementorCommon` function but Mati suggested that I will use `$e` function directly.
At the moment I think I will have to refactor all the events to the `elementorCommon` method again as, at the moment, all the events share common values.